src/main/resources/styles/app.css

```
@@ -100,38 +176,129 @@
+.rating-text {
+    -fx-text-fill: #ffd700;
+    -fx-font-weight: bold;
+    -fx-font-size: 13px;
+}
+
+/* =========================
+   VOTE COUNT BADGE
+   ========================= */
+
+.vote-count-badge {
+    -fx-background-color: rgba(255,255,255,0.08);
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)